nxcomp/src/Loop.cpp

```diff
@@ -3141,6 +3141,9 @@ int InitBeforeNegotiation()
     // Get ready to open the local display.
     //
 
+    delete xServerAddr;
+    xServerAddr = NULL;
```

Comment:
No, we have created it using 'new' so we delete it via 'delete'. In this special case this might not be necessary but more correct. If one day we use a class instead of a struct for xServerAddr we want the destructor being called.